options/locale/locale_en-US.ini

```diff
@@ -307,95 +307,95 @@ security = Security
-primary_email = Set as primary
-delete_email = Delete
-email_deletion = Delete Email
-email_deletion_desc = Deleting this email address will remove all related information from your account. Git commits using this email will remain unchanged. Do you want to continue?
-email_deletion_success = Email has been deleted successfully!
-openid_deletion = OpenID Deletion
-openid_deletion_desc = Deleting this OpenID address from your account will prevent you from signing in with it. Are you sure you want to continue ?
-openid_deletion_success = OpenID has been deleted successfully!
-add_new_email = Add new email address
-add_new_openid = Add new OpenID URI
-add_email = Add email
+primary_email = Make Primary
+delete_email = Remove
+email_deletion = Remove Email Address
+email_deletion_desc = The email address and related information will be removed from your account. Git commits by this email address will remain unchanged. Continue?
+email_deletion_success = The email address has been removed.
+openid_deletion = Remove OpenID Address
+openid_deletion_desc = Removing this OpenID address from your account will prevent you from signing in with it. Continue?
+openid_deletion_success = The OpenID address has been removed.
+add_new_email = Add New Email Address
+add_new_openid = Add New OpenID URI
+add_email = Add Email Address
 add_openid = Add OpenID URI
-add_email_confirmation_sent = A new confirmation email has been sent to '%s'. Please check your inbox within the next %s to confirm your email.
-add_email_success = Your new email address was successfully added.
-add_openid_success = Your new OpenID address was successfully added.
-keep_email_private = Keep Email Address Private
-keep_email_private_popup = Your email address will be hidden from other users if this option is set.
-openid_desc = Your OpenID addresses will let you delegate authentication to your provider of choice
+add_email_confirmation_sent = A confirmation email has been sent to '%s'. Please check your inbox within the next %s to confirm your email address.
+add_email_success = The new email address has been added.
+add_openid_success = The new OpenID address has been added.
+keep_email_private = Hide Email Address
+keep_email_private_popup = Your email address will be hidden from other users.
+openid_desc = OpenID lets you delegate authentication to an external provider.
 
 manage_ssh_keys = Manage SSH Keys
 manage_gpg_keys = Manage GPG Keys
 add_key = Add Key
-ssh_desc = These are the SSH keys associated with your account. Because these keys allow anyone using them to gain access to your repositories, it is very important you make sure you recognize them.
-gpg_desc = These are the GPG keys associated with your account. Because these keys allow commits to be verified, it is very important that you keep the corresponding private key safe.
+ssh_desc = These SSH keys are associated with your account. Make sure you recognize all of them as they allow full access to your repositories.
```

Comment:
public is also needed here then ;)

Reply:
Can you elaborate? I don't fully understand this 'public' comment.

Reply:
See line 377. It adds `public` to indicate `public GPG keys`.

Reply:
I added public to the GPG key text, since it explicitly mentioned the private keys. For consistency, I've added the public/private distinction to the SSH key text as well. Will update my branch later this week.